apps/backend/src/modules/plan-days/plan-day.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { APIPath } from "~/libs/enums/enums.js";
+import {
+	type APIHandlerOptions,
+	type APIHandlerResponse,
+	BaseController,
+} from "~/libs/modules/controller/controller.js";
+import { HTTPCode } from "~/libs/modules/http/http.js";
+import { type Logger } from "~/libs/modules/logger/logger.js";
+import { type PlanDayService } from "~/modules/plan-days/plan-day.service.js";
+import {
+	type PlanDayCreateRequestDto,
+	planDayCreateValidationSchema,
+} from "~/modules/plan-days/plan-days.js";
+
+import { PlanDaysApiPath } from "./libs/enums/enums.js";
+
+class PlanDayController extends BaseController {
+	private planDayService: PlanDayService;
+
+	public constructor(logger: Logger, planDayService: PlanDayService) {
+		super(logger, APIPath.PLAN_DAYS);
+
+		this.planDayService = planDayService;
+
+		this.addRoute({
+			handler: (options) =>
+				this.findById(options as APIHandlerOptions<{ params: { id: number } }>),
+			method: "GET",
```

Comment:
HTTP method name should be taken from the enum. There should already exist such an enum in the app